atc/metric/emitter/dogstatsd.go

```diff
@@ -31,8 +32,14 @@ func (config *DogstatsDBConfig) Description() string { return "Datadog" }
 func (config *DogstatsDBConfig) IsConfigured() bool { return config.Host != "" && config.Port != "" }
 
 func (config *DogstatsDBConfig) NewEmitter(_ map[string]string) (metric.Emitter, error) {
+	var client *statsd.Client
+	var err error
+	if config.UDS != "" {
+		client, err = statsd.New(config.UDS)
```

Comment:
I think this needs to be prefixed with `unix://`: https://github.com/DataDog/datadog-go/blob/f507940b85a2a97119a07db8be75bdf8544ead50/statsd/statsd.go#L272-L274

Given that the `--datadog-agent-uds-file` already specifies that it's for a unix domain socket, it's probably redundant to require that the user specify `unix://` there, too - so maybe we could just add the prefix here? If so, we'd want to document in the flag that it's just the filepath, without the `unix://` scheme.

Alternatively, we could rename the new flag to be more generic, e.g. `--datadog-agent-address`, and clarify that it could be prefixed with the `unix://` scheme to support UDS.

What do you think?

Reply:
Great point! When I was testing, I was prefixing `unix://` myself, but adding it as a prefix is definitely more convenient for everyone else. 

I've updated the parameter name and the description. Do you think that's sufficient or is there a way I can make it clearer?